### PR TITLE
Some fixes in AROS rom libs (graphics and intuition)

### PR DESCRIPTION
**Pull request by JuanDoble07**
BugFIX: removed limit of 64Mb (by HUNK-Size) in Amiga executable files
APICallFIX: graphics.library->ScrollRaster(rp, dx, dy, xmin, ymin, xmax, ymax) must use WORD instead of LONG in input coordinates
BugFIX: Graphics.library->ClipBlit() was using COPY area ignoring MInterm value at all in some cases.
BugFIX: Intuition/propgadgets has new support to draw labels (it wasnot implemented yet)

### CODE REVIEW

File: rom/graphics/clipblit.c [hunk 1]
```diff
@@ -183,13 +183,14 @@
         	   In this case I use a special routine to copy the rectangle
         	*/
 
-        	MoveRaster(srcRP,
+        	MoveRasteWithMInterm(srcRP,
```
Comment (MBeijer) [inline rom/graphics/clipblit.c]:
```suggestion
        	MoveRasterWithMinterm(srcRP,
```

File: rom/graphics/gfxfuncsupport.c [hunk 2]
```diff
@@ -731,13 +731,20 @@ LONG write_transp_pixels_8(struct RastPort *rp, UBYTE *array, ULONG modulo,
 
 BOOL MoveRaster (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
     	    	 WORD x2, WORD y2, BOOL UpdateDamageList, struct GfxBase * GfxBase)
+{
+    if (0 == dx && 0 == dy)
+    	return TRUE;
+
+    return MoveRasteWithMInterm (rp, dx, dy, x1, y1, x2, y2, 0xC0, UpdateDamageList, GfxBase);
```
Comment (MBeijer) [inline rom/graphics/gfxfuncsupport.c]:
```suggestion
    return MoveRasterWithMinterm (rp, dx, dy, x1, y1, x2, y2, 0xC0, UpdateDamageList, GfxBase);
```

File: rom/graphics/gfxfuncsupport.c [hunk 3]
```diff
@@ -731,13 +731,20 @@ LONG write_transp_pixels_8(struct RastPort *rp, UBYTE *array, ULONG modulo,
...
+}
+
+BOOL MoveRasteWithMInterm (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
```
Comment (MBeijer) [inline rom/graphics/gfxfuncsupport.c]:
```suggestion
BOOL MoveRasterWithMinterm (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
```

File: rom/graphics/gfxfuncsupport.h [hunk 4]
```diff
@@ -205,6 +205,9 @@ void amiga2hidd_fast(APTR src_info, OOP_Object *hidd_gc, WORD x_src , WORD y_src
 BOOL MoveRaster (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
     	    	 WORD x2, WORD y2, BOOL UpdateDamageList, struct GfxBase * GfxBase);
 
+BOOL MoveRasteWithMInterm (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
```
Comment (MBeijer) [inline rom/graphics/gfxfuncsupport.h]:
```suggestion
BOOL MoveRasterWithMinterm (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
```

File: rom/intuition/propgadgets.c [hunk 5]
```diff
@@ -444,6 +499,10 @@ void RefreshPropGadget (struct Gadget * gadget, struct Window * window,
                 if (bbox.Width <= 0 || bbox.Height <= 0)
                     break;
 
+
+			    RenderPropLabel(rp, gadget, &bbox, dri, IntuitionBase);
```
Comment (MBeijer) [inline rom/intuition/propgadgets.c]:
```suggestion
                RenderPropLabel(rp, gadget, &bbox, dri, IntuitionBase);
```

**Review by MBeijer — CHANGES_REQUESTED** [review]

**Comment by JuanDoble07** [comment]
Thanks for the review and Typo fix :)

**Comment by MBeijer** [comment]
> Thanks for the review and Typo fix :)

No problem! Otherwise I don't see anything wrong here

File: rom/graphics/gfxfuncsupport.c [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
-    Copyright � 1995-2014, The AROS Development Team. All rights reserved.
+    Copyright ? 1995-2014, The AROS Development Team. All rights reserved.
```
Comment (MBeijer) [inline rom/graphics/gfxfuncsupport.c]:
```suggestion
    Copyright © 1995-2014, The AROS Development Team. All rights reserved.
```

File: rom/intuition/propgadgets.c [hunk 7]
```diff
@@ -444,6 +499,10 @@ void RefreshPropGadget (struct Gadget * gadget, struct Window * window,
                 if (bbox.Width <= 0 || bbox.Height <= 0)
                     break;
 
+
```
Comment (MBeijer) [inline rom/intuition/propgadgets.c]:
```suggestion
```

File: rom/intuition/propgadgets.c [hunk 8]
```diff
@@ -444,6 +499,10 @@ void RefreshPropGadget (struct Gadget * gadget, struct Window * window,
...
+                RenderPropLabel(rp, gadget, &bbox, dri, IntuitionBase);
+
+
```
Comment (MBeijer) [inline rom/intuition/propgadgets.c]:
```suggestion
```

File: rom/graphics/gfxfuncsupport.c [hunk 9]
```diff
@@ -753,12 +760,12 @@ BOOL MoveRaster (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
             BltBitMap(rp->BitMap,
                       Rect.MinX + dx,
                       Rect.MinY + dy,
-	              rp->BitMap,
+                      rp->BitMap,
```
Comment (MBeijer) [inline rom/graphics/gfxfuncsupport.c:763]:
```suggestion
                      rp->BitMap,
```

File: rom/graphics/gfxfuncsupport.c [hunk 10]
```diff
@@ -753,12 +760,12 @@ BOOL MoveRaster (struct RastPort * rp, WORD dx, WORD dy, WORD x1, WORD y1,
...
                       Rect.MinX,
                       Rect.MinY,
                       Rect.MaxX - Rect.MinX + 1,
                       Rect.MaxY - Rect.MinY + 1,
-		      0xc0, /* copy */
+                      minterm,
```
Comment (MBeijer) [inline rom/graphics/gfxfuncsupport.c:768]:
```suggestion
                      minterm,
```

**Review by MBeijer — APPROVED** [review]

**Pull request merged**